src/enums/topics.enum.ts

```diff
@@ -6,5 +6,5 @@ export enum AcademicTopics {
   Git = "Git",
   Next = "Next.js",
   Solid = "Solid.js",
-  Hardware = "Hardware.cpp",
+  Hardwareacpp = "Hardware.cpp",
```

Comment:
Is character `a` in the `Hardwareacpp` typo? Maybe this could be `Hardwarecpp` or `HardwareCPP`.

Reply:
I type for more understanding that hardwareacpp = Hardware "And" C++. For easier understanding code.

Reply:
But if you want to change to "HardwareCPP" or "hardwarecpp" then you can,I have no problem with them. 👍

Reply:
So I think it could be `HardwareAndCPP`. I think add only `a` for separating the word is quite confuse to read (imagine that you're some stranger, and you see `hardw area cpp` lol). What do you think?

Reply:
Yeah,I agreed. sure we can change to HardwareAndCPP.

Reply:
changed.